reference/config_files/profiles.rst

```diff
@@ -36,7 +36,7 @@ They have this structure:
     tools.build:jobs=2
 
     [replace_requires]
-    zlib/1.2.123: zlib/*
+    zlib/1.2.123: zlib/[*]
```

Comment:
The previous syntax is supported, and it means to replace the existing version with the same version, as a place holder, but needs to be something like ``zlib/*: zlib-ng/*``, which is different than replacing something by a version range.

Reply:
```suggestion
    zlib/1.2.12: zlib/[*]
```

Reply:
> The previous syntax is supported, and it means to replace the existing version with the same version, as a place holder, but needs to be something like zlib/*: zlib-ng/*

Yes, but `zlib/*: zlib-ng/*` is documented already in https://docs.conan.io/2/reference/config_files/profiles.html#replace-requires. Nice to know, thank you.